torch/csrc/api/include/torch/data/datasets/chunk.h

```diff
@@ -12,10 +17,11 @@ namespace datasets {
 /// A chunk could be an entire file, such as an audio data file or an image,
 /// or part of a file in the case of a large text-file split based on seek
 /// positions.
-template <typename Chunk = std::vector<Example<>>>
+template <typename DataType, typename ContainerType = std::vector<DataType>>
```

Comment:
nit: it's slightly better to call this `ExampleType_` or something like that, because this name shows up in error messages. It's good to keep it aligned with the typedef inside the class.

Reply:
I will change DataType to ExampleType_ and COntainerType to ChunkType_